app/components/login/page.jsx

```diff
@@ -0,0 +1,93 @@
+"use client"; 
+import React, { useState } from "react";
+import { Button, TextField } from "@mui/material";
+import "./page.css";
+
+export default function Login(props) {
+
+  const [userName, setUserName] = useState('')
+  const [password, setPassword] = useState('')
+
+
+  // Ham check cac chu viet hoa tra ve true
+  const containsUppercase = (str) => {
+    return /[A-Z]/.test(str);
+  }
+
+
+  // Ham check ky tu dac biet
+  const checkSpecialSymbol = (str) => {
+    return /[ `!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?~]/.test(str)
+  }
+
+
+  // ham check space
+  const checkSpace = (str) => {
+    return /[ ]/.test(str);
+  }
+
+
+  // ham check validation cua username
+  const userValidation = () => {
+    if (containsUppercase(userName) || checkSpecialSymbol(userName) || userName.length === 0) {
+      return true
+    }
+    if (!containsUppercase(userName) && !checkSpecialSymbol(userName) || !userName.length === 0) {
+      return false
+    }
+  }
+
+
+  //ham check validation cua password
+  function passValida() {
+    if (password.length === 0 || checkSpace(password)) {
+      return true;
+    }
+    else {
+      return false;
+    }
+  }
+
+
+  //ham handle click login button
+  const submitLogin = (e) => {
+    e.preventDefault()
+  }
+
+  return (
+    <div className="container_login">
```

Comment:
As we already have "Box" component in material UI
You should use "Box" instead of "div"

Reply:
confirm